modules/administration-guide/pages/uninstalling-che.adoc

```diff
@@ -20,26 +20,16 @@ To uninstall an instance of {prod} {prod-ver}:
 
 .Procedure
 
-. Obtain the name of the {prod-short} {orch-namespace}. The default is `{prod-namespace}`.
+* Remove the {prod-short} instance from the `{prod-namespace}` {orch-namespace}:
```

Comment:
I suggest getting rid of `{prod-namespace}` because it might not be necessarily the case (some customers might be using a different namespace), plus it's not important to mention here.
```suggestion
* Remove the {prod-short} instance:
```